cfme-performance/utils/ssh.py

```
@@ -152,7 +155,7 @@ def run_command(self, command, timeout=RUNCMD_TIMEOUT, reraise=False, log_less=F
     def run_rails_command(self, command, timeout=RUNCMD_TIMEOUT):
         logger.info("Running rails command `%s`", command)
         return self.run_command('cd /var/www/miq/vmdb; bin/rails runner {}'.format(command),
-            timeout=timeout)
+            timeout=timeout, ignore_stderr=True)
```

adding a comment here specifying the reason for including this, would be helpful. Just a mentioned of the issue like this: 
```# https://github.com/redhat-performance/cfme-performance/issues/4 ```

Also, ignoring the error is only required for getting the config for yaml_data as per #4 
So running this for all rails command isn't fair. Don't you think? I would further channelize this to https://gist.github.com/hroyrh/56363764f0b1ec5ab058b6e59a37cef1#file-scannererror-L36 if possible.